lab_5_scrapper/scrapper.py

```diff
@@ -176,6 +330,23 @@ def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> No
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        header = article_soup.find('h1')
+        if header:
+            self.article.title = header
+
+        self.article.author = 'NOT FOUND'
+
+        element = article_soup.find('ul', class_='list-unstyled list-inline').find('li', class_='list-inline-item')
+        date = element.get_text().strip().split(', ')[1]
+        self.article.date = date
+
+        meta_tag = article_soup.find('meta', attrs={'meta': 'keywords'})
+        topics = []
```

Comment:
topic is needed for 8, do you plan for 8?

Reply:
yes